KioskTests/ListingsViewControllerTests.swift

```diff
@@ -19,39 +19,44 @@ class ListingsViewControllerConfiguration: QuickConfiguration {
                 subject.loadViewProgrammatically()
             }
 
+            /*
+            Note that we use a tolerance of 1% to account for antialiasing issues that differ
+            from machine to machine.
+            */
```

Comment:
Interesting, the old technique used to be binary bit for bit, I imagine this must be quite a bit slower - but it's not exactly a test suite we run too often 👍